ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TransactionMeta.scala

```diff
@@ -19,11 +20,25 @@ import com.daml.lf.data.{Ref, Time}
   *   and to traffic-shape the work handled by DAML applications
   *   communicating over the ledger.
   *
+  * @param submissionTime: the transaction submission time
+  *
+  * @param submissionSeed: The seed used to derive the transaction contract IDs.
+  *
+  * @param optUsedPackages: the set of package IDs the transaction is depending on.
+  *   undefined is not known. Undefined is not known.
```

Comment:
Description is repeated, I would suggest rephrasing it a bit:
```suggestion
  *   Undefined means 'not known'.
```